src/rules/comment-syntax.ts

```diff
@@ -155,7 +154,7 @@ export default createRule<Options, MessageIds>({
               context.report({ node: comment, messageId: 'lineCommentCapital' });
             }
 
-            if (lastChar === '.') {
+            if (lastChar === '.' && !comment.value.endsWith('etc.') && !comment.value.endsWith(' ...')) {
```

Comment:
Why are the 3 dots preceeded with a space? Can't I have something like `Foo bar...`? Or what is the use case for?

Reply:
Maybe add some tests for these new use cases?

Reply:
Yea I guess we can allow your example as well if we want. And I can add some tests, good catch. Thought I added them but prolly forgot then.

Reply:
Ok updated the rule to allow your case as well and only check if it ends with `...`.